cartographer/io/pbstream_main.cc

```diff
@@ -26,14 +26,15 @@ int main(int argc, char** argv) {
   google::InitGoogleLogging(argv[0]);
 
   FLAGS_logtostderr = true;
-  google::SetUsageMessage(
+  const std::string usage_message =
```

Comment:
Interestingly this worked on my machine.

Reply:
hmm... maybe you have a different version that allows to set it twice